src/style.css

```diff
@@ -1,3 +1,63 @@
 body {
   margin: 0;
 }
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  text-transform: uppercase;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.logo {
+  justify-content: space-between;
```

Comment:
The `justify-content` property is not applicable to inline elements like `.logo`. Consider removing it or changing the display property to a flex container if needed.